src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,101 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int DEFAULT_CAPACITY = 16;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        size = 0;
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        double threshold = table.length * DEFAULT_LOAD_FACTOR;
+        if (size == threshold) {
```

Comment:
is there chances it can be true? since you are comparing double and int using ==